lambdas/services/create_document_reference_service.py

```diff
@@ -103,6 +102,15 @@ def create_document_reference_request(
                 )
```

Comment:
Can't comment those lines, but could we move the check on line 98 for existing LG records to line 65?

Reply:
Hi, for this one, I think the check on line 98 might be better to be after LG filename checks on line 97,
The reason is that:
1. the check will make API call to dynamodb table, so better to run all locally computed validation first
2. at the end of the check in line 98, it will remove incomplete upload records

So I am leaning towards keeping it on line 98, but I will rename the function at line 98 `check_existing_lloyd_george_records` --> `check_existing_lloyd_george_records_and_remove_failed_upload` so that the function name better reflects what it does.